python/src/trezorlib/cardano.py

```diff
@@ -654,22 +673,22 @@ def get_native_script_hash(
 
 
 def sign_tx(
-    client,
+    client: "TrezorClient",
     signing_mode: messages.CardanoTxSigningMode,
     inputs: List[InputWithPath],
     outputs: List[OutputWithAssetGroups],
     fee: int,
     ttl: Optional[int],
     validity_interval_start: Optional[int],
-    certificates: List[CertificateWithPoolOwnersAndRelays] = (),
-    withdrawals: List[messages.CardanoTxWithdrawal] = (),
+    certificates: List[CertificateWithPoolOwnersAndRelays] = [],
```

Comment:
don't use `[]` as a default value. pylint will correctly shout at you.

we might want to change protobuf generator to use `Sequence[...]` in place of `List[...]` so that sending tuples to protobuf is valid.

Reply:
Done in 3c5b9b6 - the default now is empty tuple again, just some helper functions needed to be modified to eat `Sequence` instead of `List`. None of these were actually inputted into protobuf message directly, only their length